doc/installation.rst

```diff
@@ -4,71 +4,50 @@ Installation
-
-* `Anaconda <https://www.anaconda.com/download/>`_
-* `WinPython <https://winpython.github.io/>`_
-* `Python(x,y) <https://code.google.com/p/pythonxy>`_
-
-You can start using it immediately after installing one of them (you only need
-to install one!).
-
-
-The hard way
-~~~~~~~~~~~~
-
-If you want to install Spyder directly, you need to follow these steps:
-
-#. Install the essential requirements:
-
-   * `The Python programming language <http://www.python.org/>`_
-   * `PyQt5 <http://www.riverbankcomputing.co.uk/software/pyqt/download5>`_ (recommended)
-     or `PyQt4 <http://www.riverbankcomputing.co.uk/software/pyqt/download>`_
-
-#. Install Spyder and its dependencies by running this command::
-
-       pip install spyder
-
+This section explains how to install the latest stable release of
+Spyder. If you prefer testing the development version, please use
+the `bootstrap` script (see next section).
 
-Updating Spyder
-~~~~~~~~~~~~~~~
 
-You can update Spyder by:
+The Easy/Recommended Way: Anaconda
+----------------------------------
 
-* Updating Anaconda, WinPython or Python(x,y).
+Thanks to the Spyder team and `Anaconda <https://www.anaconda.com/>`_, Spyder is
+included in the `Anaconda <https://www.anaconda.com/download/>`_ Python distribution,
+which comes with everything you need to get started in an all-in-one package.
 
-* Or using this command (in case you *don't* use any of those scientific
-  distributions)::
+This is the easiest way to install Spyder for any of our platform, and
+the way we recommend to avoid unexpected issues we aren't able to
+help you with. If in doubt, you should install via this method;
+it generally has the least likelihood of potential pitfalls for non-experts,
+and we may be able to provide limited assistance if you do run into trouble.
 
-        pip install --upgrade spyder
 
-  .. note::
+The Harder Way: Alternative Distributions
+----------------------------------------------------
 
-     This command will also update all Spyder dependencies
-
-|
+**Important Note:** While we offer alternative options for users who
+desire them, if you encounter installation issues you are unable to
+solve on your own, we recommend you use Anaconda instead, as we are
+generally unable to offer individual assistance for problems specific
+to installing via these alternative distributions.
 
-Installing on macOS
--------------------
+Windows
+~~~~~~~
 
-The easy way
-~~~~~~~~~~~~
+Spyder is included in these other scientific Python distributions:
 
-Thanks to the Spyder team and `Anaconda <https://www.anaconda.com/>`_, Spyder is included
-in the `Anaconda <https://www.anaconda.com/download/>`_ Python distribution,
-which comes with everything you need to get started in an all-in-one package.
+* `WinPython <https://winpython.github.io/>`_
+* `Python(x,y) <https://code.google.com/p/pythonxy>`_
 
+You can start using it immediately after installing one of them
+(you only need to install one!), just like with Anaconda.
```

Comment:
These lines are not needed after removing the Python(x,y) link.

Reply:
Yes; removed already.